mozilla-services/consvc-shepherd
# [SHEP-167] Remove DB seeder script

## References

- [SHEP-167](https://mozilla-hub.atlassian.net/browse/SHEP-167)

## Problem Statement

The seed command script was initially added to generate random data in our database so it could facilitate testing of models during development. However, as we created new models and updated existing ones, the seed script hasn't been kept up to date. Also, the Boostr sync script has become faster and more efficient, so the seed script is no longer as necessary as initially anticipated.

## Proposed Changes

- Remove the seed command script and related documentation.
- Make minor documentation fixes identified during the review.

## Verification Steps

Confirm that there are no remaining mentions or uses of the seed script.



[SHEP-167]: https://mozilla-hub.atlassian.net/browse/SHEP-167?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
